src/notification/send-notification.spec.ts

```diff
@@ -0,0 +1,64 @@
+import axios from "axios";
+import { NotificationType } from "./notification.type";
+import { NotificationChannel } from "./channels";
+import { UserNotificationTopic } from "./topics";
+import SpyInstance = jest.SpyInstance;
+import { sendNotifications } from "./send-notification";
+
+jest.mock("axios");
+
+describe("Send Notification", () => {
+  const port = 303030;
+  const clientName = "test";
+  let notification: NotificationType | NotificationType[];
+  let spy: SpyInstance;
+
+  describe("Sending notification", () => {
+    describe("Multiple notifications", () => {
+      beforeEach(async () => {
+        spy = jest.spyOn(axios, "post").mockResolvedValue({});
+        notification = {
+          topic: UserNotificationTopic.USER_ONBOARDED,
+          channel: "sms",
+          payload: {
+            to: "1234",
+            from: "1234",
+            message: "Hello",
+          },
+        };
+        await sendNotifications(clientName, port, [notification, notification]);
+      });
+
+      it("should call axios once", () => {
+        expect(spy).toBeCalledTimes(2);
```

Comment:
Is this meant to call axios once or 2 times

Reply:
The function takes a notification or an array of notifications